js/binance.js

```diff
@@ -5295,4 +5296,58 @@ module.exports = class binance extends Exchange {
         }
         return result;
     }
+
+    async fetchBorrowInterestAccrued (code = undefined, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        let market = undefined;
+        if (code !== undefined) {
+            const currency = this.currency (code);
+            request['asset'] = currency['id'];
+        }
+        if (since !== undefined) {
+            request['startTime'] = since;
+        }
+        if (limit !== undefined) {
+            request['size'] = limit;
+        }
+        if (symbol !== undefined) { // Isolated
+            market = this.market (symbol);
+            request['isolatedSymbol'] = market['id'];
+        }
+        const response = await this.sapiGetMarginInterestHistory (this.extend (request, params));
+        //
+        // {
+        //     "rows":[
+        //         {
+        //             "isolatedSymbol": "BNBUSDT", // isolated symbol, will not be returned for crossed margin
+        //             "asset": "BNB",
+        //             "interest": "0.02414667",
+        //             "interestAccuredTime": 1566813600000,
+        //             "interestRate": "0.01600000",
+        //             "principal": "36.22000000",
+        //             "type": "ON_BORROW"
+        //         }
+        //     ],
+        //     "total": 1
+        // }
+        //
+        const rows = this.safeValue (response, 'rows');
+        const interest = [];
+        for (let i = 0; i < rows.length; i++) {
+            const row = rows[i];
+            const timestamp = this.safeNumber (row, 'interestAccuredTime');
+            interest.push ({
+                'account': this.safeString (row, 'isolatedSymbol', 'CROSS'), // isolated symbol, will not be returned for crossed margin
```

Comment:
We shouldn't really return raw market ids to the user in the unified structures, gotta convert it to a unified symbol before returning it.

Reply:
right

Reply:
@samgermain lmk if you're going to fix that

Reply:
I will, if it's the weekend I'll usually make fixes after the weekend, my timezone is also GMT-6